### PR TITLE
Add 'Create Object' and 'Create Object Type' functions to client

### PR DESCRIPTION
**Pull request by jdeb9**

### CODE REVIEW

File: src/admin-client.ts [hunk 1]
```diff
@@ -71,6 +71,41 @@ export default class PIPAdminClient {
...
+        json,
+      }),
+    });
+    this.verifyResponse(resp);
+    return resp.json();
+  };
+
+  createObjectType = async <T>(
+    name: string,
+    children: string[],
```
Comment (elsimir) [inline src/admin-client.ts]: These should probably be optional, seeing as you're already assigning them an empty array below to handle that case and its nice that users don't have to supply them.

Also probably reverse the order of children and parents as its far more likely when creating them that you know the parent but not the children.

File: src/types.ts [hunk 2]
```diff
@@ -18,6 +18,17 @@ export interface PIPObject<T = object> {
   json: T;
 }
 
+export interface PIPObjectType<T = object> {
```
Comment (elsimir) [inline src/types.ts]: This does not need, or use T.

Comment (elsimir) [reply src/types.ts]: Also this is the same type as ObjectType above in this file, it doesn't need to be redeclared

**Comment by jdeb9** [comment]
@elsimir requested changes have been pushed

**Pull request merged**